From: Andrew Morton <akpm@linux-foundation.org>
Date: Sat, 31 Jan 2026 12:29:16 -0800
Subject: [PATCH] 
 maple_tree-start-using-maple-copy-node-for-destination-checkpatch-fixes

ERROR: code indent should use tabs where possible
#492: FILE: lib/maple_tree.c:3167:
+^I^I^I^I^I         slot_max, ma_pivots(mn, mt),$

ERROR: code indent should use tabs where possible
#493: FILE: lib/maple_tree.c:3168:
+^I^I^I^I^I         ma_slots(mn, mt));$

total: 2 errors, 0 warnings, 721 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

NOTE: Whitespace errors detected.
      You may wish to use scripts/cleanpatch or scripts/cleanfile

./patches/maple_tree-start-using-maple-copy-node-for-destination.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: "Liam R. Howlett" <Liam.Howlett@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 lib/maple_tree.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/maple_tree.c b/lib/maple_tree.c
index e0929bf0cfa1a..4d9e7f00f5c80 100644
--- a/lib/maple_tree.c
+++ b/lib/maple_tree.c
@@ -3164,8 +3164,8 @@ static inline void cp_dst_to_slots(struct maple_copy *cp, unsigned long min,
 		if (mt_is_alloc(mas->tree)) {
 			if (ma_is_leaf(mt)) {
 				cp->gap[d] = ma_leaf_max_gap(mn, mt, slot_min,
-					         slot_max, ma_pivots(mn, mt),
-					         ma_slots(mn, mt));
+						 slot_max, ma_pivots(mn, mt),
+						 ma_slots(mn, mt));
 			} else {
 				unsigned long *gaps = ma_gaps(mn, mt);
 
-- 
2.47.3

